templates/patterns/pagination/pattern-pagination.html.twig

```diff
@@ -4,18 +4,45 @@
  * Pagination pattern.
  */
 #}
+
+{% set internal_items = [] %}
+
+{% for item in items %}
+  {% set internal_items = internal_items|merge([item|merge({
+      'attributes': item.attributes ?
+                    item.attributes.setAttribute('aria-label', item.aria_label|default(item.label))
+                    : create_attribute().setAttribute('aria-label', item.aria_label|default(item.label)),
+    })])
```

Comment:
A more concise version is:
```suggestion
      'attributes': create_attribute({'aria-label': item.label}).merge(item.attributes|default(create_attribute()))
    })])
```
First we create a new attribute object with the default value, then we merge it to the existing one (so we don't override items that already exist). In case there is no existing one, we create it.

Reply:
Indeed, thank you. I was thinking on how to make it smaller and your suggestion was apprecieted. I changed all them accordingly.